runtime/runtime-core/jvm/src/aws/smithy/kotlin/runtime/smoketests/SmokeTestsFunctionsJVM.kt

```diff
@@ -0,0 +1,6 @@
+package aws.smithy.kotlin.runtime.smoketests
+
+import kotlin.system.exitProcess
+
+public actual fun exitProcess(status: Int): Nothing = exitProcess(status)
+public actual fun getEnv(name: String): String? = System.getenv(name)
```

Comment:
**Question**: This seems like [platform functionality](https://github.com/smithy-lang/smithy-kotlin/blob/main/runtime/runtime-core/common/src/aws/smithy/kotlin/runtime/util/Platform.kt) to me—getting environment variables is already something we do cross-platform and process control seems like a natural addition. Is there any reason we can't reuse `PlatformProvider` for env vars and add support for exiting the process with a return code?

Reply:
Thanks for calling this out, I almost reinvented the wheel here